uwblueprint/jump-math
# fix: add graphql date scalar type

## Notion ticket link

[Add GraphQL Date scalar type](https://www.notion.so/uwblueprintexecs/Add-GraphQL-Date-scalar-type-496d25d288944ff5850f72964797e705)



## Implementation description
* Add scalar type using built-in GraphQL scalar plumbing on backend
* Add scalar type using `apollo-link-scalars` library on frontend



## Steps to test
1. Load the teacher test session page and verify it still looks good data-wise.
2. Inspect the network request if you like to verify that it's a number being sent over-the-wire.



## What should reviewers focus on?
* Any structural changes?


## Checklist
